Gems/PhysX/Code/Editor/KinematicWidget.cpp

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Editor/KinematicWidget.h>
+#include <Editor/KinematicDescriptionDialog.h>
+
+#include <AzFramework/Physics/PropertyTypes.h>
+#include <AzToolsFramework/API/ToolsApplicationAPI.h>
+
+namespace PhysX
+{
+    namespace Editor
+    {
+        AZ::u32 KinematicWidget::GetHandlerName() const
+        {
+            return Physics::Edit::KinematicSelector;
+        }
+
+        QWidget* KinematicWidget::CreateGUI(QWidget* parent)
```

Comment:
As mentioned above, if we updated the existing combo box handler to support an edit button, then we wouldn't need to introduce an entirely new handler.

Reply:
As we took a while to get here I'd rather add this as-is for now and then see if we can refactor/unify after if possible